python_modules/dagster/dagster/_core/execution/backfill.py

```diff
@@ -49,6 +49,8 @@ class PartitionBackfill(
             ("backfill_timestamp", float),
             ("error", Optional[SerializableErrorInfo]),
             ("asset_selection", Optional[Sequence[AssetKey]]),
+            ("title", Optional[str]),
+            ("description", Optional[str]),
```

Comment:
We also talked about adding a description field to the backfill. This seems less necessary, but is a low lift to add. 

Options:
1. dont add the description field
2. add the description field but don't expose it in the UI yet
3. add the description field to the UI

I don't have a clear use case in mind where the description adds a lot of value, maybe that's a sign it's unnecessary.

Reply:
I think it'd be alright to add this but I agree we don't have an ideal example of how it'd be used... maybe we allow markdown here (just in the UI), and things creating backfills can use this to show links to other systems or a "table of git commit messages" or something related to the intent behind the backfill?

Reply:
discord was pretty excited about having a longer form description when  we met with them yesterday, so i think this is worth keeping for them! they launch a lot of backfills based on their own system they built and thought the long form description would be really useful for them to document why their system launched a particular backfill

Reply:
we've chatted with some users who think having a description field would be useful, so i'm planning to keep it!